### PR TITLE
Remove forced redirects support

### PR DESCRIPTION
**Pull request by justindotpub**
- Remove forced redirects support
- Return a human-readable error is someone attempts to use forced redirects

### CODE REVIEW

**Comment by justindotpub** [comment]
Closes https://github.com/ipfs-shipyard/go-ipfs-redirects/issues/4

File: redirects.go [hunk 1]
```diff
@@ -118,13 +114,13 @@ func ParseString(s string) ([]Rule, error) {
...
 
-// parseStatus returns the status code and force when "!" suffix is present.
-func parseStatus(s string) (code int, force bool, err error) {
+// parseStatus returns the status code.
+func parseStatus(s string) (code int, err error) {
 	if strings.HasSuffix(s, "!") {
-		force = true
-		s = strings.Replace(s, "!", "", -1)
+		// See https://docs.netlify.com/routing/redirects/rewrites-proxies/#shadowing
+		return -1, fmt.Errorf("forced redirects (or \"shadowing\") are not supported by IPFS gateways")
```
Comment (justindotpub) [inline redirects.go:122]: I avoided "aka" and thought "also known as" might be long for an error message.  Including links seems bad and prone to needing updating.  I think I personally would prefer just "forced redirects" but if "shadowing" helps people, great. 👍

**Comment by justindotpub** [comment]
In the interest of having everything ready for you to approve and merge, I'm going to go ahead and merge this so that I can get my latest changes in kubo pointing at this code.  But let me know if you want anything changed here.

**Review by lidel — COMMENTED** [review]
lgtm, thank you!

**Pull request merged**